requirements.txt

```diff
@@ -0,0 +1,6 @@
+Flask==0.10.1
+itsdangerous==0.24
```

Comment:
I produced the requirements file like this:
$ pip freeze > requirements.txt
As far as I'm concerned, that is standard procedure. I don't see any reason to do otherwise.